docs/rfcs/archival_storage_service.md

```diff
@@ -0,0 +1,59 @@
+# Archival Storage Service
+
+## Problem Statement
+
+In order to provide archivists with reliable long term digital storage of both digitised and "born-digital" assets we need a service that can: 
+
+- Ensure safe long term storage assets of digitised assets
+- Follow industry practises around file integrity and audit trails
+- Provide a scalable mechinsm for indetifying and retrieving content
+
+## Suggested Solution
+
+We propose to build a storage service based on AWS S3 and DynamoDb. 
+
+![archival storage service - page 1](https://user-images.githubusercontent.com/953792/39753255-0987d2fe-52b6-11e8-97ee-633ac09e1a9e.png)
+
+### Process flow
+
+- Work is uploaded in gzipped BagIt format to S3 to an "Ingest Bucket"
+- The S3 event stream triggers an AWS Lambda, translating the event to an SNS notification
+- The "Archival Service" picks up work as an [SQS Autoscaling Service](https://github.com/wellcometrust/terraform-modules/tree/master/sqs_autoscaling_service) subscribed to the topic.
+- The Archival Service performs the following operations:
+  - Obtain a filestream of the uploaded compressed artefact from its ingest location
+  - Stream the decompressed artefact to a processing location
```

Comment:
I think practically this breaks down into a number of smaller services:
- Decompress
- Format check
- Move to long term storage

Though whether we choose to implement it that way or not is up for grabs.